rllib/policy/policy.py

```diff
@@ -65,16 +65,18 @@ class you pass into RolloutWorker will be constructed with
 
     @abstractmethod
     @DeveloperAPI
-    def compute_actions(self,
-                        obs_batch,
-                        state_batches=None,
-                        prev_action_batch=None,
-                        prev_reward_batch=None,
-                        info_batch=None,
-                        episodes=None,
-                        explore=None,
-                        timestep=None,
-                        **kwargs):
+    def compute_actions(
+            self,
+            obs_batch=None,  # TODO: deprecate
+            state_batches=None,  # TODO: deprecate
+            prev_action_batch=None,  # TODO: deprecate
+            prev_reward_batch=None,  # TODO: deprecate
+            info_batch=None,  # TODO: deprecate
+            episodes=None,  # TODO: deprecate
+            explore=None,
+            timestep=None,
+            trajectories=None,  # TODO: move this up as the main input source
```

Comment:
Instead of modifying the old compute actions method, can you instead create a new one that takes in a single trajectory_view argument?

That way we retain perfect backwards compatibility. The default implementation can call the old compute actions method.